test/unit/utils.test.js

```diff
@@ -456,4 +457,25 @@ suite(Support.getTestDialectTeaser('Utils'), function() {
       expect(stack[3].getFunctionName()).to.eql('this_here_test');
     });
   });
+
+  suite('Sequelize.cast', function() {
+    var sql = Support.sequelize;
+    var generator = sql.queryInterface.QueryGenerator;
+    var run = generator.handleSequelizeMethod.bind(generator);
+    var expectsql = Support.expectsql
+
+    if (Support.getTestDialect() !== 'mssql') {
```

Comment:
No need to skip mssql - It doesn't need to be 100 percent correct syntax (in my mind), as long as it tests the functionality (that you can pass nested conditions)